cmd/admin-heal.go

```diff
@@ -133,6 +135,30 @@ func (s stopHealMessage) JSON() string {
 	return string(stopHealJSONBytes)
 }
 
+// backgroundHealStatusMessage is container for stop heal success and failure messages.
+type backgroundHealStatusMessage struct {
+	Status   string `json:"status"`
+	HealInfo madmin.BgHealState
+}
+
+// String colorized to show background heal status message.
+func (s backgroundHealStatusMessage) String() string {
+	healPrettyMsg := console.Colorize("HealBackgroundTitle", "Background healing status:\n")
+	healPrettyMsg += fmt.Sprintf("  Total items scanned: %s\n",
+		console.Colorize("HealBackground", s.HealInfo.ScannedItemsCount))
+	healPrettyMsg += fmt.Sprintf("  Last background heal check: %s\n",
+		console.Colorize("HealBackground", timeDurationToHumanizedDuration(time.Since(s.HealInfo.LastHealActivity)).String()+" ago"))
+	return healPrettyMsg
+}
+
+// JSON jsonified stop heal message.
+func (s backgroundHealStatusMessage) JSON() string {
```

Comment:
s.Status should have "success"

Reply:
It does @harshavardhana 

```
21:42 $ go install && mc --json admin heal myminio/
{
 "status": "success",
 "HealInfo": {
  "ScannedItemsCount": 227,
  "LastHealActivity": "2019-06-26T20:42:43.723099077Z"
 }
}
```

Success is set in line 203